plugins/calibre.koplugin/search.lua

```diff
@@ -189,8 +187,8 @@ local CalibreSearch = InputContainer:new{
         "find_by_authors",
         "find_by_path",
     },
-    user_libraries = DataStorage:getSettingsDir() .. "/" .. libraries_file,
-    user_book_cache = DataStorage:getSettingsDir() .. "/" .. metadata_file,
+    user_libraries = DataStorage:getDataDir() .. "/cache/" .. libraries_file,
+    user_book_cache = DataStorage:getDataDir() .. "/cache/" .. metadata_file,
```

Comment:
Just mentionning this, to make sure you're ok with it:
https://github.com/koreader/koreader/blob/c1be488a11bc9efb3082b77affe990e189fbdbe6/frontend/apps/filemanager/filemanagermenu.lua#L241-L256
So, this menu item (that people seldom use I guess) will trash your calibre caches too.

Reply:
I'm totally fine with the books cache. It gets redone each time:
- we scan calibre libraries
- a single calibre library has a newer timestamp than the cache

About the libraries cache I don't know. I think most users will use just one calibre library per mountpoint. If they're using calibre device drivers the metadata will be at the root of each mountpoint and the scan will finish in a few (milli)seconds.

If the user has a lot of calibre libraries under a remote path `/mnt/onboard/.books/managed/calibre/libraries` **and** a lot of files in between the scan could take a couple of minutes. The worst case scenario is on desktop, where we scan the whole home dir. Maybe it makes sense to keep libraries under settings. Pinging @NiLuJe

Reply:
I mean, it does say "Clear cache" and the Calibre setting says "Cache stuff", so, err, I don't see the problem ;).

Reply:
(i.e., I totally agree that cache is where that belongs).